src/rancilio-pid.cpp

```diff
@@ -659,7 +615,52 @@ void refreshTemp() {
         }
     }
 }
+boolean sys = readSysParamsFromStorage();
```

Comment:
why move all of this down?

Reply:
To make sure that the SysParams are loaded before the scale is initiated.